app/client/src/features/dashboard/DashboardEvent.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import * as React from 'react';
+import { ListItemText } from '@mui/material';
+import { formatDate } from '@local/utils/format';
+
+interface DashboardEventProps {
+    id: string,
+    title: string,
+    description: string,
+    startDateTime: Date,
+    organization: string,
+}
+
+export function DashboardEvent(props: DashboardEventProps) {
```

Comment:
I think this should be giving you a lint warning? I'm not actually 100% sure and it's not a big deal if you're not getting one. In most components, we destructure the props;

`export function DashboardEvent({ id, title, description, startDateTime, organization }: DashboardEventProps) {`